tutorials/ios-chatbot/part-3/index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: How to build a conversational app using Cloud Machine Learning APIs (Part 3 of 3)
+description: How to build a conversational app using Cloud Machine Learning APIs (Part 3 of 3).
+author: PokerChang
+tags: Cloud Functions, Dialogflow, API.AI, Webhooks, Localization, Chatbot, Machine Learning API, Transation, Vision, Speech
+date_published: 2017-01-02
+---
+
+In [part 1] and [part 2] of this series, we showed you how to build a conversational tour guide app with API.AI and Google Cloud Machine Learning APIs. In this final part, you'll learn how to extend this app to the Google Assistant-supported devices (Google Home, eligible Android phones and iPhones, and Android Wear). And we'll build this on top of the existing API.AI agent created in parts 1 and 2.
+
+[![The Google Assistant / Google Home Demo](http://img.youtube.com/vi/_x5rlkpZiyc/0.jpg)](https://youtu.be/_x5rlkpZiyc)
+
+### <a name="actions"></a>New Intents for Actions on Google
+
+In [part 1], we discussed the app's input and output context relationships. 
+
+
+![alt_text](conversational-app-p3-3.png "Contexts without the Assistant")
+
+
+The _where_ context requires the user to upload an image, which is not supported by the Google Assistant. We can modify the context relationship as below. 
+
+![alt_text](conversational-app-p3-8.png "Contexts with the Assistant")
+
+We will add three new intents, _hours-no-context_, _ticket-no-context_ and _map-no-context_. Each intent will set _location_ as the output context so that other intents can use the _location_ as an input parameter.
+
+![alt_text](conversational-app-p3-5.png "Contexts Screenshot")
+
+### Enable Actions on Google Integration
+
+Now we'll enable Actions on Google to support the Google Assistant.
+
+1. Open your API.AI console. Under the _Integrations_ Tab, turn on the _Actions on Google_ integration.
+
+![alt_text](conversational-app-p3-1.png "Enable Actions on Google Integration")
+
+2. In the popup dialog under _Additional triggering intents_, add all intents  you want to support on the Google Assistant. The system will automatically set the _Welcome Intent_ to _Default Welcome Intent_. You can also click _SETTINGS_ under Actions on Google to bring up this settings dialog in the future. Note that the _inquiry.where_ intent requires uploading an image and won't work on the Google Assistant, so you should not  add that intent to the triggering intents list. We discussed how to add a new intent to support that in 
+
+[New Intent for Actions on Google](#heading=actions) section.
+
+3. After you're done adding all the intents that we want to support on Google on Actions (e.g hours-no-context intent) to the _additional triggering intents_ list, hit _UPDATE _and _TEST_ button on the bottom. It will generate a green box. Tap the _VIEW_ button to go to the Actions on Google Web Simulator.
```

Comment:
In the first sentence, not sure if the _UPDATE _and _TEST_ button is correct. Is it a single button called _UPDATE and TEST_, or 2 buttons, _UPDATE_ and _TEST_ ?